doc/develop/modules.rst

```diff
@@ -495,13 +495,64 @@ module:
      cmake: .
      kconfig: Kconfig
 
+.. _sysbuild_module_integration:
+
+Sysbuild integration
+====================
+
+:ref:`Sysbuild<sysbuild>` is the Zephyr build system that allows for building
+multiple images as part of a single application, the sysbuild build process
+can be extended externally with modules as needed, for example to add custom
+build steps or add additional targets to a build. Inclusion of
+sysbuild-specific build files, :file:`CMakeLists.txt` and :file:`Kconfig`, can
+be described as:
+
+.. code-block:: yaml
+
+   build:
+     sysbuild-cmake: <cmake-directory>
+     sysbuild-kconfig: <directory>/Kconfig
+
+The ``sysbuild-cmake: <cmake-directory>`` part specifies that
+:file:`<cmake-directory>` contains the :file:`CMakeLists.txt` to use. The
+``sysbuild-kconfig: <directory>/Kconfig`` part specifies the Kconfig file to
+use.
+
+Here is an example :file:`module.yml` file referring to
+:file:`CMakeLists.txt` and :file:`Kconfig` files in the `sysbuild` directory of
+the module:
+
+.. code-block:: yaml
+
+   build:
+     sysbuild-cmake: sysbuild
+     sysbuild-kconfig: sysbuild/Kconfig
+
+The module description file :file:`zephyr/module.yml` can also be used to
+specify that the build files, :file:`CMakeLists.txt` and :file:`Kconfig`, are
+located in a :ref:`modules_module_ext_root`.
+
+Build files located in a ``MODULE_EXT_ROOT`` can be described as:
+
+.. code-block:: yaml
+
+   build:
+     sysbuild-cmake-ext: True
+     sysbuild-kconfig-ext: True
+
+This allows control of the build inclusion to be described externally to the
+Zephyr module.
+
 Build system integration
 ========================
 
 When a module has a :file:`module.yml` file, it will automatically be included into
 the Zephyr build system. The path to the module is then accessible through Kconfig
 and CMake variables.
 
+Normal modules
```

Comment:
Shouldn't we keep calling those Zephyr modules ?
That's the name we have been using so far and match the variable names below.

```suggestion
Zephyr modules
```